advanced_source/cpp_export.rst

```diff
@@ -92,28 +92,33 @@ vanilla Pytorch model::
 
 Because the ``forward`` method of this module uses control flow that is
 dependent on the input, it is not suitable for tracing. Instead, we can convert
-it to a ``ScriptModule`` by subclassing it from ``torch.jit.ScriptModule`` and
-adding a ``@torch.jit.script_method`` annotation to the model's ``forward``
-method::
+it to a ``ScriptModule``. 
+In order to convert the module to the ``ScriptModule``, one needs to 
+enable ``torch.jit._enable_recursive_script`` 
```

Comment:
This flag is gone in nightly and recursive mode is always on

Reply:
oopsie, I'll fix this.